src/Apps/Auction/Components/BidForm.tsx

```diff
@@ -33,7 +35,11 @@ const validationSchema = Yup.object().shape({
   ),
 })
 
-export const BidForm: React.FC<Props> = ({ onSubmit, saleArtwork }) => {
+export const BidForm: React.FC<Props> = ({
+  onSubmit,
+  saleArtwork,
+  showPricingTransparency = false,
```

Comment:
I'm not trying to suggest here that this prop should be permanent, just wanted an easy toggle for it while developing in storybook and staging. Open to other ideas!